src/test/java/pl/javastart/streamstask/ExampleTest.java

```diff
@@ -1,6 +1,135 @@
 package pl.javastart.streamstask;
 
+import org.junit.Test;
+import org.junit.jupiter.api.BeforeEach;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.math.BigDecimal;
+import java.util.*;
+import java.util.stream.Collectors;
+
 public class ExampleTest {
 
+    @BeforeEach
+    public List<User> takeUserList() {
+        List<User> users = new ArrayList<>();
+
+        users.add(new User(1L, "Alicja", 20));
+        users.add(new User(2L, "Dominik", 15));
+        users.add(new User(3L, "Patrycja", 25));
+        users.add(new User(4L, "Marcin", 30));
+        users.add(new User(5L, "Tomek", 18));
+        users.add(new User(6L, "Damian", 26));
+
+        return users;
+    }
```

Comment:
Podzieliłbym tę metodę na tworzenie mężczyzn i kobiet i później
```
List<User> users = new ArrayList<>();
users.addAll(women());
users.addAll(men());
```
będzie to przydatne w samych testach, bo pozwala łatwo pobrać rozmiar, czy kolekcję samych kobiet, czy mężczyzn